bundle/src/main/java/com/adobe/acs/commons/mcp/form/FieldComponent.java

```diff
@@ -133,7 +133,7 @@ public String getName() {
 
     public boolean hasOption(String optionName) {
         return Stream.of(formField.options())
-                .filter(s -> s.equalsIgnoreCase(optionName))
+                .filter(s -> s.equalsIgnoreCase(optionName) || s.startsWith(optionName + "="))
```

Comment:
@badvision does this look OK?

Reply:
Yes, absolutely!  Thanks.  That was a future area of expansion and the next logical step for that bit.  I didn't like having to use a string array but the fact is that annotations provide you very few options so it was a necessary evil in order to pass less-structured data along.